modules/gql/schema.graphql

```diff
@@ -20,6 +20,13 @@ enum TransactionStatus {
   PROCESSED
 }
 
+enum Asset {
+  HIVE
+  HIVE_CONSENSUS
+  HBD
+  HBD_SAVINGS
```

Comment:
Values are lowercase in the DB
```suggestion
hive
hive_consensus
hbd
hbd_savings
```

Reply:
Resolved in https://github.com/vsc-eco/go-vsc-node/pull/22/commits/2649d1224559a18637c300d7c43af5571a3f97f0